modules/geo/src/main/java/org/opensearch/geo/search/aggregations/metrics/AbstractGeoBoundsAggregator.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geo.search.aggregations.metrics;
+
+import org.opensearch.common.lease.Releasables;
+import org.opensearch.common.util.BigArrays;
+import org.opensearch.common.util.DoubleArray;
+import org.opensearch.search.aggregations.Aggregator;
+import org.opensearch.search.aggregations.InternalAggregation;
+import org.opensearch.search.aggregations.metrics.MetricsAggregator;
+import org.opensearch.search.aggregations.support.ValuesSource;
+import org.opensearch.search.aggregations.support.ValuesSourceConfig;
+import org.opensearch.search.internal.SearchContext;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Abstract class for doing the {@link GeoBounds} Aggregation over fields of type GeoShape and GeoPoint.
```

Comment:
```suggestion
 * Abstract class for doing the {@link GeoBounds} Aggregation over fields of type geo_shape and geo_point.
```

Reply:
what is the benefit for geo_shape and geo_point here?

Reply:
Sure, if it is not useful, we can remove it. Suggestion is if we are adding explicitly, lets call field type with right case.

Reply:
I will update it, as it is a nit-pick